### PR TITLE
add backend API routes

### PR DESCRIPTION
**Pull request by kevintsoii**
from #12

### CODE REVIEW

File: server/src/router/v1/events.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    const eventID = c.req.param('eventID');
+    return c.json({ eventID });
+});
+
+events.get('/:eventID/files', c => {
+    const eventID = c.req.param('eventID');
+    return c.json({ eventID });
+});
+
+export default events;
```
Comment (knhn1004) [inline server/src/router/v1/events.ts]: add empty line

File: server/src/router/v1/equipment.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    const equipmentTypeID = c.req.param('equipmentTypeID');
+    return c.json({ equipmentTypeID });
+});
+
+equipment.get('/equipmentitem/:equipmentItemID/rental-history', c => {
+    const equipmentItemID = c.req.param('equipmentItemID');
+    return c.json({ equipmentItemID });
+});
+
+export default equipment;
```
Comment (knhn1004) [inline server/src/router/v1/equipment.ts]: add empty line

File: server/src/router/v1/company.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    const companyID = c.req.param('companyID');
+    return c.json({ companyID });
+});
+
+company.get('/:companyID/subscribers/count', c => {
+    const companyID = c.req.param('companyID');
+    return c.json({ companyID });
+});
+
+export default company;
```
Comment (knhn1004) [inline server/src/router/v1/company.ts]: add empty line

File: server/src/router/v1/projects.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    const projectID = c.req.param('projectID');
+    return c.json({ projectID });
+});
+
+projects.get('/:projectID/files', c => {
+    const projectID = c.req.param('projectID');
+    return c.json({ projectID });
+});
+
+export default projects;
```
Comment (knhn1004) [inline server/src/router/v1/projects.ts]: empty line

File: server/src/router/v1/officers.ts [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import { Hono } from 'hono';
+
+const officers = new Hono();
+
+officers.get('/', c => c.json({}));
+
+export default officers;
```
Comment (knhn1004) [inline server/src/router/v1/officers.ts]: empty line

File: server/src/router/v1/majors.ts [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+const majors = new Hono();
+
+majors.get('/', c => c.json({}));
+
+majors.get('/:majorName/users', c => {
+    const majorName = c.req.param('majorName');
+    return c.json({ majorName });
+});
+
+export default majors;
```
Comment (knhn1004) [inline server/src/router/v1/majors.ts]: empty line

File: server/src/router/v1/users.ts [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+usersRouter.get('/', async c => {
+    const foundUsers: User[] = await db.select().from(users);
+	return c.json({
+		users: foundUsers,
+	});
+});
+
+usersRouter.get('/rental-history', c => c.json({}));
+
+export default usersRouter;
```
Comment (knhn1004) [inline server/src/router/v1/users.ts]: empty line

**Review by knhn1004 — CHANGES_REQUESTED** [review]

**Review by ManasC478 — DISMISSED** [review]

**Review by knhn1004 — APPROVED** [review]

**Review by knhn1004 — CHANGES_REQUESTED** [review]
halt for now, we drastically changed the route structure

**Pull request closed without merging**